src/main/java/orion/chatbot/Ui.java

```diff
@@ -0,0 +1,197 @@
+    }
+
+    /**
+     * Prints the greeting message and logo to the console.
+     */
+    public void printGreet() {
+        printBar();
+        System.out.println(Ui.LOGO);
+        printBar();
+
+        printIndent("Hello from Orion!");
+        printIndent("We're fetching your task list from the cosmos...");
+        printBar();
+    }
+
+    /**
+     * Prints the goodbye message to the console.
+     */
+    public void printGoodbye() {
+        printIndent("Bye! Hope to see you again soon!");
+    }
+
+    /**
+     * Reads a command from the user input.
+     *
+     * @return The command input by the user.
+     */
+    protected String readCommand() {
+        String input = sc.nextLine();
+        printBar();
+        return input;
+    }
+
+    /**
+     * Prints the list of tasks to the console.
+     *
+     * @param taskDescriptions A list of task descriptions to be printed.
+     */
+    public void printTaskList(List<String> taskDescriptions) {
+        printIndent("Here are the tasks in your list:");
+        for (String s : taskDescriptions) {
+            printIndent(s);
+        }
+    }
+
+    /**
+     * Prints a message indicating that a task has been added to the task list.
+     *
+     * @param tasks The {@code TaskList} to which the task has been added.
+     * @param task  The task that was added.
+     */
+    public void printAddTask(TaskList tasks, Task task) {
+        printIndent("Sure! I've added the following task to your list:");
+        printIndent(task.toString());
+        printIndent("Now you have " + tasks.getNoTasks() + " tasks in your list.");
+    }
+
+    /**
+     * Prints a message indicating that a task has been deleted from the task list.
+     *
+     * @param tasks The {@code TaskList} from which the task has been deleted.
+     * @param task  The task that was deleted.
+     */
+    public void printDeleteTask(TaskList tasks, Task task) {
+        printIndent("Sure! I've deleted the following task:");
+        printIndent(task.toString());
+        printIndent("Now you have " + tasks.getNoTasks() + " tasks in your list.");
+    }
+
+    /**
+     * Prints a message indicating that a task has been marked as done.
+     *
+     * @param task The task that was marked as done.
+     */
+    public void printMarkTask(Task task) {
+        printIndent("Sure! I've marked the following task as done:");
+        printIndent(task.toString());
+    }
+
+    /**
+     * Prints a message indicating that a task has been marked as undone.
+     *
+     * @param task The task that was marked as undone.
+     */
+    public void printUnmarkTask(Task task) {
+        printIndent("Sure! I've marked the following task as undone:");
+        printIndent(task.toString());
+    }
+
+    /**
+     * Prints a message indicating the number of tasks loaded from storage.
+     *
+     * @param taskNo The number of tasks loaded.
+     */
+    protected void printLoadTasks(int taskNo) {
```

Comment:
Please make the method name a little less ambiguous, as printLoadTasks can be interpreted as a Task might be printed.